src/entry-dev.js

```diff
@@ -5,18 +5,14 @@ import { Provider } from 'react-redux';
 import { init } from './Store';
 import App from './App';
 import logger from 'redux-logger';
-import { NotificationsPortal } from '@red-hat-insights/insights-frontend-components/components/Notifications';
```

Comment:
Do not remove `NotificationsPortal` othwerwise you will loose notifications.

Reply:
gonna revert, cuz no reason to do too much here